colelandolt/personal-website
# Add Tests for field validation

## Description

- Adds `active` field to the `Subscriber` model
- Updates the `email` field of the `Subscriber` model to require unique entries
- Creates tests to validate entries to the `Subscriber` model

## Validation

- `python manage.py makemigrations`
- `python manage.py migrate`
- `python manage.py createsuperuser`
- `python manage.py runserver`
- Submit new entry in the form on the home page
- Log into the Django Admin
- Check that the entry was stored in the database
- `python manage.py dumpdata pages > pages/fixtures/subscribers.json`
- `python manage.py dumpdata auth.User > pages/fixtures/users.json`
- `python manage.py test`
